index.js

```diff
@@ -535,18 +550,19 @@ class Plugin extends MiniAccountsPlugin {
 
     // sign a claim
     const clientChannel = account.getClientChannel()
-    const encodedClaim = util.encodeClaim(newBalance.toString(), clientChannel)
+    const newDropBalance = util.xrpToDrops(this.baseToXrp(newBalance))
+    const encodedClaim = util.encodeClaim(newDropBalance.toString(), clientChannel)
     const keyPairSeed = util.hmac(this._secret, CHANNEL_KEYS + account.getAccount())
     const keyPair = nacl.sign.keyPair.fromSeed(keyPairSeed)
     const signature = nacl.sign.detached(encodedClaim, keyPair.secretKey)
 
-    debug(`signing outgoing claim for ${newBalance.toString()} drops on ` +
+    debug(`signing outgoing claim for ${newDropBalance.toString()} drops on ` +
       `channel ${clientChannel}`)
 
     const aboveThreshold = new BigNumber(util
       .xrpToDrops(account.getClientPaychan().amount))
```

Comment:
Not part of this PR, but shouldn't the client paychan's amount be reloaded after funding?

Reply:
can we put this as an issue?

Reply:
Fixed in #30